hasher-matcher-actioner/docs/docs/installation/index.md

```diff
@@ -1 +1,90 @@
 # Quick install guide
+
+This guide allows you to install a `released` version of HMA without any customizations to the source code. 
+
+## Pre-requisites
+
+1. [Docker](https://www.docker.com/)
+2. [Terraform cli](https://www.terraform.io/)
+3. [aws cli](https://aws.amazon.com/cli/) Configured
+4. [jq](https://stedolan.github.io/jq/)
+## Mirror HMA Releases to your private ECR repository
+
+HMA uses docker images to deploy code to your infrastructure. This requires the images to be available within your private cloud account. We use AWS ECR [Elastic Container Registry](https://aws.amazon.com/ecr/).
+
+HMA Releases are deployed to a public ECR repository [https://gallery.ecr.aws/l5b3f6x2/hma/releases](https://gallery.ecr.aws/l5b3f6x2/hma/releases).
+
+### Creating a private ECR repository mirroring our releases
+
+This assumes your AWS access keys allow you to create an ECR Repository.
+
+```shell
+$ aws ecr create-repository --repository-name hma-releases-mirror
+```
+
+### Copying a release to your private ECR repository
+
+```shell
+$ export HMA_RELEASE="v0.1.1"
```

Comment:
will need to change this soon (I just found a bug with v0.1.1 so want to remove that version)